.changeset/lucky-rocks-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/hooks": minor
```

Comment:
Totally my fault I didn't mention this earlier, but could you please move it to the main `"frontity"` package?